packages/ui/src/components/token/TokenSummary.tsx

```diff
@@ -28,8 +30,38 @@ const TokenSummary: FC<{
 }
 
 const Balances = ({ children }: { children: React.ReactNode }) => {
+    const state = useBlankState()!
+
+    const isLoading =
+        state.isNetworkChanging || state.isRatesChangingAfterNetworkChange
+
+    const balancesLoadingSkeleton = (
```

Comment:
Can you export this to a new component?

Reply:
For me is easier to read this component in this way:


```javascript

return (
<>
{ isLoading ? <BalanceLoadingSkeleton />  : < div>{children} </div> }
</>
)

```

Reply:
Make sense cleaner that way!